cpp/src/ray/runtime/object/native_object_store.cc

```diff
@@ -60,6 +72,11 @@ std::vector<std::shared_ptr<msgpack::sbuffer>> NativeObjectStore::GetRaw(
   std::vector<std::shared_ptr<msgpack::sbuffer>> result_sbuffers;
   result_sbuffers.reserve(results.size());
   for (size_t i = 0; i < results.size(); i++) {
+    auto meta = results[i]->GetMetadata();
+    if (meta != nullptr) {
+      std::string meta_str((char *)meta->Data(), meta->Size());
+      CheckException(meta_str);
```

Comment:
Good! Can you add some description about Exception in api.h? Users should know which api will try exceptions.